README.md

```diff
@@ -43,3 +43,30 @@ package main
 var File_hello_txt = []byte("Hello\r\n")
 ```
 
+Also, you could use directories as `PATH`. For example: 
+
+```bash
+$ mkdir -p configs
+$ echo 'config1' > configs/1.conf
+$ echo 'config2' > configs/2.conf
+$ ./embd-go -p "configs" configs
+$ cat embd/data.go
+// DO NOT EDIT BY HAND
+//
+// Generated with:
+//
+//	embd-go "-p" "configs" "configs"
+
+package configs
+
+var Dir_configs = struct {
+	File_1_conf []byte
```

Comment:
Hmpf, sorry, it showed as unindented on my browser. I verified in an Other Browser&trade; that it's ok, thanks.